src/main/java/devybot/tasks/Task.java

```diff
@@ -0,0 +1,34 @@
+package devybot.tasks;
+public class Task {
+    private String description;
+    private boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public void markTask() {
+        this.isDone = true;
+    }
+
+    public void unmarkTask() {
+        this.isDone = false;
+    }
+
+    public boolean checkDone() {
```

Comment:
Should this instead by isDone() or isMarked()? so that it sounds more like a boolean